src/GitHub.TeamFoundation.14/Home/GitHubHomeSection.cs

```diff
@@ -21,22 +24,59 @@ namespace GitHub.VisualStudio.TeamExplorer.Home
     public class GitHubHomeSection : TeamExplorerSectionBase, IGitHubHomeSection
     {
         public const string GitHubHomeSectionId = "72008232-2104-4FA0-A189-61B0C6F91198";
-        IVisualStudioBrowser visualStudioBrowser;
+        const string TrainingUrl = "https://services.github.com/on-demand/windows/visual-studio";
+        const string DocsUrl = "https://visualstudio.github.com/docs";
+
+        readonly IVisualStudioBrowser visualStudioBrowser;
+        readonly ITeamExplorerServices teamExplorerServices;
+        readonly IPackageSettings settings;
+        readonly IUsageTracker usageTracker;
 
         [ImportingConstructor]
         public GitHubHomeSection(IGitHubServiceProvider serviceProvider,
-            ISimpleApiClientFactory apiFactory, ITeamExplorerServiceHolder holder,
-            IVisualStudioBrowser visualStudioBrowser)
+            ISimpleApiClientFactory apiFactory,
+            ITeamExplorerServiceHolder holder,
+            IVisualStudioBrowser visualStudioBrowser,
+            ITeamExplorerServices teamExplorerServices,
+            IPackageSettings settings,
+            IUsageTracker usageTracker)
             : base(serviceProvider, apiFactory, holder)
         {
             Title = "GitHub";
             View = new GitHubHomeContent();
             View.DataContext = this;
             this.visualStudioBrowser = visualStudioBrowser;
+            this.teamExplorerServices = teamExplorerServices;
+            this.settings = settings;
+            this.usageTracker = usageTracker;
 
             var openOnGitHub = ReactiveCommand.Create();
             openOnGitHub.Subscribe(_ => DoOpenOnGitHub());
             OpenOnGitHub = openOnGitHub;
+
+            // We want to display a welcome message but only if Team Explorer isn't
+            // already displaying the "Install 3rd Party Tools" message. To do this
+            // we need to set a timer and check in the tick as at this point the message
+            // won't be initialized.
+            if (!settings.HideTeamExplorerWelcomeMessage)
+            {
+                var timer = new DispatcherTimer();
+                timer.Interval = new TimeSpan(10);
+                timer.Tick += (s, e) =>
+                {
+                    timer.Stop();
+                    if (!IsGitToolsMessageVisible())
+                    {
+                        ShowWelcomeMessage();
+                    }
+                };
+                timer.Start();
+            }
+        }
+
+        bool IsGitToolsMessageVisible()
+        {
+            return teamExplorerServices.IsNotificationVisible(new Guid("DF785C7C-8454-4836-9686-D1C4A01D0BB9"));
```

Comment:
Probably a good idea to put this and other GUID next to the other GUIDs we have in the Guids class, so they're easy to keep track of